index.js

```diff
@@ -111,6 +111,28 @@ Babel.prototype.optionsHash = function() {
       hash[key] = (typeof value === 'function') ? (value + '') : value;
     }
 
+    if (options.plugins) {
+      hash.plugins = [];
```

Comment:
Most likely, we want to tell babel-tranpsiler to:

1. load a plugin name X relative to Y
2. provide it a mechanism to instantiate and configure that plugin, where the outputs of the config are automatically serialized and used to derive a cache key. If the serialization is is not possible, we should warn/throw/disable the cache.